theano/tensor/nnet/nnet.py

```diff
@@ -384,6 +384,7 @@ def c_code(self, node, name, inp, out, sub):
             }
         }
 
+        #pragma omp parallel for schedule(static, 32)
```

Comment:
Hi nouiz,

Thanks for your reply.
**1. I do the timing by using gcc and icc. Both of them. 
2. My OS are Ubuntu and Centos. Both of them works.
3. I'll try inherit from OpenMPOp and update the diff of PR.**

Thanks a lot.